mne/viz/evoked.py

```diff
@@ -1063,7 +1063,8 @@ def _plot_evoked_white(evoked, noise_cov, scalings=None, rank=None, show=True):
     for idx in passive_idx[::-1]:  # reverse order so idx does not change
         evoked.del_proj(idx)
 
-    evoked.pick_types(meg=True, eeg=True, ref_meg=False, exclude='bads')
+    evoked.pick_types(meg=True, eeg=True, seeg=True, ecog=True,
+                      ref_meg=False, exclude='bads')
```

Comment:
We should probably have a `PICK_DATA_KWARGS` constant so this can just be:
```
evoked.pick_types(**PICK_DATA_KWARGS)
```
And hopefully `_pick_data_channels` is just `pick_types(info, **PICK_DATA_KWARGS)` or so

Reply:
agreed